Code/GraphMol/MolDraw2D/Wrap/rdMolDraw2D.cpp

```diff
@@ -134,7 +134,7 @@ void pyListToColourVec(python::object pyo, std::vector<DrawColour> &res) {
 }  // namespace
 
 void pyDictToMapColourVec(python::object pyo,
-                           std::map<int, std::vector<DrawColour> > &res) {
+                          std::map<int, std::vector<DrawColour>> &res) {
```

Comment:
Sorry, I betray my age.  gcc used to insist on spaces between > in these circumstances.

Reply:
Used to be required. Now it's optional and our clang-format setup removes the spaces.